modules/virt-creating-single-pvc-template-storage-pool.adoc

```diff
@@ -4,74 +4,71 @@
 
 :_content-type: PROCEDURE
 [id="virt-creating-single-pvc-template-storage-pool_{context}"]
-= Creating a storage pool using a pvcTemplate specification in a host path provisioner (HPP) custom resource.
+= Creating a storage pool for hostpath provisioner volumes
```

Comment:
I realize this was like this to begin with, but this should really be split into a concept module and a procedure module. The intro for the procedure module should simply describe what the user is going to do in the procedure.

Reply:
Oh yeah! And the creating storage class section above also needs to be split into two modules. :-(